sources/common/src/XmlSignalFactory.cpp

```diff
@@ -70,106 +70,104 @@ PARSER::XmlDescription::XmlDescription(const pugi::xml_node &rNode) :
     }
 }
 
-const std::string & PARSER::XmlDescription::GetRegex() const
+const std::string &PARSER::XmlDescription::GetRegex() const
 {
-    if (m_RegexLine.length())
+    if (!m_RegexLine.empty())
     {
         return m_RegexLine;
     }
     throw EXCEPTION::VcdException(EXCEPTION::Error::NO_REGEX_EXPRESSION,
-        "XML - No parsing regular expression.");
+                                  "XML - No parsing regular expression.");
 }
 
-const std::string & PARSER::XmlDescription::GetTimestamp() const
+const std::string &PARSER::XmlDescription::GetTimestamp() const
 {
     if (m_Timestamp.length())
```

Comment:
Oh I omitted this one. To check if `std::string` is not empty use `!m_Timestamp.empty()`. This is more readable. Also in lines below in every other function `GetXXX`.